core/src/main/scala/com/karumi/shot/domain/model.scala

```diff
@@ -17,9 +17,9 @@ object model {
 object Config {
 
   val androidDependencyMode: FilePath = "androidTestImplementation"
-  val androidDependencyGroup: String = "com.facebook.testing.screenshot"
-  val androidDependencyName: String = "core"
-  val androidDependencyVersion: String = "0.12.0"
+  val androidDependencyGroup: String = "com.karumi"
+  val androidDependencyName: String = "shot-android"
+  val androidDependencyVersion: String = "4.0.1-SNAPSHOT"
```

Comment:
What if......

![what if](https://media.giphy.com/media/23cl0y02AdTURPWhgR/giphy.gif)

we replace this version with "4.+" or directly any version so the user will be able to consume any artifact published without updating the gradle plugin? If we go for the "4.+" version we would be able to control breaking changes :smiley:

This would be awesome. Imagine we add a new feature to the interface....just by publishing the shot-android artifact, any user would be able to consume it without updating the whole plugin. The downside is....fixing a bug would also be a breaking change 🤔 What do you think guys?

Reply:
I am not completely sure about this. Even if we limit the range to only minor version I would be concerned that if there is a mistake in a release, users might get bugs/errors without making any change.

Reply:
We can keep it as is for the first release and we'll see how the project evolves 😃